src/components/Common/ButtonText.tsx

```
@@ -0,0 +1,130 @@
+  const getColorStyle = () => {
+    switch (variant) {
+      case ButtonTextVariant.PRIMARY:
+        return 'text-status-blue-300';
+      case ButtonTextVariant.ALTERNATIVE:
+      default:
+        return 'text-text-alternative';
+    }
+  };
+
+  // 버튼 타입에 따라 다른 강도의 시각적 피드백을 주기 위해 투명도를 매핑
+  const getPressStrength = (): PressStrength => {
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)